driver-core/src/main/com/mongodb/client/model/geojson/Point.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.client.model.geojson;
+
+import static com.mongodb.assertions.Assertions.notNull;
+
+/**
+ * A representation of a GeoJSON Point.
+ *
+ * @since 3.1
+ */
+public final class Point extends Geometry {
```

Comment:
There are two Point classes in the JDK: java.awt.Point and javafx.scene.effect.Point.  Neither are likely to be used in the same Java file along with the Java driver.